python/lsst/obs/hsc/hscMapper.py

```diff
@@ -84,23 +91,28 @@ def __init__(self, **kwargs):
         #
         self.filters = {}
         for f in [
-            "W-S-G+",
-            "W-S-R+",
-            "W-S-I+",
-            "W-S-Z+",
-            "W-S-ZR",
             "HSC-G",
             "HSC-R",
             "HSC-I",
             "HSC-Z",
             "HSC-Y",
             "ENG-R1",
+            "NB0515",
+            "NB0527",
+            "NB0656",
+            "NB0718",
+            "NB0816",
+            "NB0921",
+            "IB0945",
+            "NB0973",
             "SH",
             "PH",
             "NONE",
             "UNRECOGNISED"]:
             # Get the canonical name -- see #2113
             self.filters[f] = afwImage.Filter(afwImage.Filter(f).getId()).getName()
+        self.defaultFilterName = "UNRECOGNISED"
```

Comment:
Discussed this with @laurenam; we agreed that it's ok to merge this, on the understanding that we still have to merge the rest of HSC-987 on a different (LSST) ticket and that it'll be easy to backtrack and take care of this if it causes problems in DM-2984.